storage/database/interface.go

```diff
@@ -77,6 +80,7 @@ type Batch interface {
 	Write() error
 	// Reset resets the batch for reuse
 	Reset()
+	Close()
```

Comment:
Please leave a comment that this is needed only if the database is DynamoDB.

Reply:
I reverted the change.